StepLang.CLI/FormatCommand.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System.Drawing;
+using Pastel;
+using StepLang.Tokenizing;
+
+namespace StepLang.CLI;
+
+internal static class FormatCommand
+{
+    private static VerbosityWriter stdout = new(Console.Out, Verbosity.Normal);
+    private static VerbosityWriter stderr = new(Console.Error, Verbosity.Normal);
+
+    public static async Task<int> Invoke(string[] filesOrDirs, bool setExitCode, bool dryRun, Verbosity verbosity)
+    {
+        stdout = new(Console.Out, verbosity);
+        stderr = new(Console.Error, verbosity);
+
+        if (dryRun)
+            await stdout.Normal("Dry run mode enabled. No files will be modified.");
+
+        if (filesOrDirs.Length == 0)
+            filesOrDirs = new [] { "." };
+
+        var changes = 0;
+        var files = 0;
+        foreach (var s in filesOrDirs)
+        {
+            var (c, f) = await Format(s, dryRun);
+            changes += c;
+            files += f;
+        }
+
+        await stdout.Normal($"{(dryRun ? "Would have applied" : "Applied")} {changes} change(s) to {files} file(s).");
```

Comment:
Maybe use Humanizer.Pluralize here

Reply:
Not sure there is much of a benefit here. Of the top of my head, I can't think of any other messages/communication that would benefit from it. I think its better to keep the number of external dependencies low if possible.

Reply:
>Not sure there is much of a benefit here.

Per se, no, but you'll have more and more compiler messages eventually. Anyway, just a thought.